source/includes/_invoice.md

```diff
@@ -6011,7 +6011,7 @@ If successful, returns a status code of 200 and a list of all accounts.
 
 ### Search invoices
 
-Search for an account by a specified search string. If the search string is a number, it is compared to the `invoiceNumber` attribute. An exact match is required. Otherwise, it is compared to the following attributes: `invoiceId`, `accountId`, or `currency`. The operation returns all account records in which the search string matches all or part of any one of these attributes.
+Search for an account by a specified search string. If the search string is a number, it is compared to the `invoiceNumber` attribute. An exact match is required. Otherwise, it is compared to the following attributes: `invoiceId`, `accountId`, or `currency`. The operation returns all account records in which the search string matches all or part of any one of these attributes. All `search` apis for a specific type (e.g `Invoice`) return the shallow objects without doing any computation. So for instance, the amount, or balance would require pulling more data such the list of `InvoiceItem`, and as a result such fields are not populated (and default to 0).
```

Comment:
The sentence I gave you about `All search apis for a specific type (e.g Invoice) ...` was to explain to you the general behavior we have. However, I don't think it makes sense to copy it verbatim:
1. Instead, because this section is about `Invoice`, ket's **only** talk about invoice here, and explain what is being returned and not returned.
2. Most resources (e.g Account, Invoice, Payment, ...) will offer such search api, so I would expect this PR to also document such endpoints similarly.

Reply:
Hi @reshmabidikar Yes at the time of adding details about all the search and list endpoints, I had checked these ones as well. Payment method and Bundles do not have any such fields and the Payment ones return the amount for each payment. So these should be ok.